### DISCUSSION TITLE
Re bill my account

### DISCUSSION THREAD

**Discussion by AyabongaQwabi** [community/community#43731] [Copilot Conversations]
Hi I previously had no funds in my account and payment declined, nowI have funds but my account is locked from doing another transaction. I would like to re activate github copilot and there's no way to talk directly to support can you help

**Comment by melvincwng** [comment]
@AyabongaQwabi, you can raise a ticket & contact GitHub Support directly over [here](https://support.github.com/contact/account).

Hope this helps you in your account related issues 😄
